clouddriver-kubernetes/src/test/groovy/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationPropertiesSpec.groovy

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.config
+
+import com.google.common.collect.ImmutableMap
+import com.netflix.spinnaker.fiat.model.Authorization
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class KubernetesConfigurationPropertiesSpec extends Specification {
+
+  @Unroll
+  void "two accounts' properties are properly bound to KubernetesConfigurationProperties"() {
+    when:
+    def ACC_0_NAME = "k8s-acc-0"
+    def ACC_0_READ_0 = "acc-0-read-0-role"
+    def ACC_0_READ_1 = "acc-0-read-1-role"
+    def ACC_0_WRITE_0 = "acc-0-write-0-role"
+    def ACC_0_WRITE_1 = "acc-0-write-1-role"
+    def ACC_0_CACHE_THREADS = 2
+    def ACC_0_NS_0 = "acc-0-ns-0"
+    def ACC_0_NS_1 = "acc-0-ns-1"
+    def ACC_0_OMIT_KIND_0 = "clusterRoleBinding"
+    def ACC_0_OMIT_KIND_1 = "customResourceDefinition"
+    def ACC_0_KUBECONFIG = "/tmp/config"
+    def ACC_0_CRD_0_K8S_KIND = "deploymentConfig"
+    def ACC_0_CRD_0_NAMESPACED = false
+    def ACC_0_CRD_1_K8S_KIND = "Route"
+    def ACC_0_CRD_1_VERSIONED = true
+    def ACC_1_NAME = "k8s-acc-1"
+    def ACC_1_READ_0 = "acc-1-read-0-role"
+    def ACC_1_READ_1 = "acc-1-read-1-role"
+    def ACC_1_WRITE_0 = "acc-1-write-0-role"
+    def ACC_1_WRITE_1 = "acc-1-write-1-role"
+    def ACC_1_CACHE_THREADS = 3
+    def ACC_1_OMIT_NS_0 = "acc-1-ns-0"
+    def ACC_1_OMIT_NS_1 = "acc-1-ns-1"
+    def ACC_1_KIND_0 = "role"
+    def ACC_1_KIND_1 = "apiService"
+    def ACC_1_KUBECONFIG = "/tmp/config1"
+    def ACC_1_CRD_0_K8S_KIND = "deploymentConfig"
+    def ACC_1_CRD_0_NAMESPACED = false
+    def ACC_1_CRD_1_K8S_KIND = "Route"
+    def ACC_1_CRD_1_VERSIONED = true
+    def map = new ImmutableMap.Builder()
+      .put("kubernetes.accounts[0].name", ACC_0_NAME)
```

Comment:
Can you add `rawResourcesEndpointConfig` section as well?

Reply:
added `rawResourcesEndpointConfig` section is added in the unit test case.